### PR TITLE
h2spec 2.6.0 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/h2spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  def install
+    commit = Utils.safe_popen_read("git", "rev-parse", "--short", "HEAD").chomp
+    ldflags = %W[
+      -s -w
+      -X main.VERSION=#{version}
+      -X main.COMMIT=#{commit}
+    ]
+    system "go", "build", *std_go_args, "-ldflags", ldflags.join(" "), "./cmd/h2spec"
+    # (bin/"h2spec").chmod 555
```
Comment (SeekingMeaning) [inline Formula/h2spec.rb:20]: ?

Comment (chenrui333) [reply Formula/h2spec.rb]: 😂

File: Formula/h2spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  url "https://github.com/summerwind/h2spec.git",
+    tag:      "v2.6.0",
+    revision: "70ac2294010887f48b18e2d64f5cccd48421fad1"
+  license "MIT"
+  head "https://github.com/summerwind/h2spec.git"
+
+  depends_on "go" => :build
+
+  def install
+    commit = Utils.safe_popen_read("git", "rev-parse", "--short", "HEAD").chomp
```
Comment (SeekingMeaning) [inline Formula/h2spec.rb:13]: I might see if I could add a `git_commit` helper function or something, but that's not blocking for this PR

**Review by SeekingMeaning — APPROVED** [review]

File: Formula/h2spec.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/h2spec.rb:13]: second to this idea.

Also might be good to include helper function for shell completions as well

**Pull request closed without merging**